From: Petr Tesarik <ptesarik@suse.com>
Subject: Make sure VIDEO_CAPABILITY_64BIT_BASE is defined
Upstream: never, build fix for SLE12

SLE12 did not provide a definition for VIDEO_CAPABILITY_64BIT_BASE
in <linux/screen_info.h>.

Signed-off-by: Petr Tesarik <ptesarik@suse.com>
---
 kexec/arch/i386/x86-linux-setup.c |    4 ++++
 1 file changed, 4 insertions(+)

--- a/kexec/arch/i386/x86-linux-setup.c
+++ b/kexec/arch/i386/x86-linux-setup.c
@@ -37,6 +37,10 @@
 #include "x86-linux-setup.h"
 #include "../../kexec/kexec-syscall.h"
 
+#ifndef VIDEO_CAPABILITY_64BIT_BASE
+#define VIDEO_CAPABILITY_64BIT_BASE     (1 << 1)        /* Frame buffer base is 64-bit */
+#endif
+
 void init_linux_parameters(struct x86_linux_param_header *real_mode)
 {
 	/* Fill in the values that are usually provided by the kernel. */
